packages/Search/src/DTK_SearchMacros.hpp

```diff
@@ -0,0 +1,17 @@
+/****************************************************************************
+ * Copyright (c) 2012-2019 by the DataTransferKit authors                   *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the DataTransferKit library. DataTransferKit is     *
+ * distributed under a BSD 3-clause license. For the licensing terms see    *
+ * the LICENSE file in the top-level directory.                             *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#ifndef DTK_SEARCH_MACROS_HPP
+#define DTK_SEARCH_MACROS_HPP
+
+#define DTK_MARK_REGION( x ) std::string( "DTK_" ) + x
+
+#endif
```

Comment:
I'd rather have it in `details/` for now

Reply:
Could do that.